test/unit/_suite.ts

```diff
@@ -17,6 +19,21 @@ export default suite<Utils, any>((tests) => {
   tests({
     expect,
     spy: (...args) => (<any>sandbox.spy)(...args),
-    stub: (...args) => (<any>sandbox.stub)(...args)
+    stub: (...args) => (<any>sandbox.stub)(...args),
+    configurable: (clazz) => {
```

Comment:
I think these would be better named `itShouldBeConfigurable()` and `itShouldHaveAlias()`